components/integration/ballerina-vs-apollo-for-graphql/code/apollo-graphql-bbe/designer-tool.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Built-in designer tool for GraphQL"
-description: "Explore the boundless possibilities with the in-built GraphQL API designer, a visual tool of the Ballerina VS Code plugin. Effortlessly design and prototype GraphQL APIs, unlocking a seamless and intuitive development experience. Empower your GraphQL services with this exceptional visual designer tool."
+description: "Explore the built-in GraphQL API designer, a visual tool in the Ballerina VS Code plugin. Easily design and prototype GraphQL APIs for a smooth development experience. Improve your GraphQL services with this visual tool."
```

Comment:
```suggestion
description: "Explore the built-in GraphQL API designer, a visual tool made available via the Ballerina VS Code plugin. Easily design and prototype GraphQL APIs for a smooth development experience. Improve your GraphQL services with this visual tool."
```

Reply:
Not entirely sure about the last sentence though. Does it really improve services?